recipe/build.sh

```diff
@@ -1,5 +1,11 @@
 #!/bin/bash
 
+## manual patch
+## NB: This should be removed in the next release!
+bzip2 -d inst/tests/tests.Rraw.bz2
+patch --verbose -V none inst/tests/tests.Rraw.bz2 < ${RECIPE_DIR}/patches/0001-no-warn-old-dates.patch
+bzip2 inst/tests/tests.Rraw
```

Comment:
Is this just a patch formatting issue or something?

Reply:
CRAN preprocesses some source files to compressed formats, such that the file the patch applies to needs decompression first. Please let me know If there is a more idiomatic way to deal with that.

The error is just a typo - I meant to apply the patch to the plain file.

Reply:
Ah ok! Did not realize this. Thanks for explaining Mervin 🙏

Not that I'm aware of. Do we track this in an upstream conda-build and/or rattler-build issue?

Reply:
> Do we track this in an upstream conda-build and/or rattler-build issue?

Not that I know of - this is the first I ever hit it. I doubt it happens often enough to merit reworking the generic patching strategy upstream. Most R packages never do this and it is limited to rarely accessed files (here a massive unit test collection).